csrc/mmdeploy/utils/opencv/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ mmdeploy_add_library(${PROJECT_NAME} opencv_utils.cpp)
 
 target_link_libraries(${PROJECT_NAME}
         PRIVATE mmdeploy::core
-        PUBLIC ${OpenCV_LIBS})
+        PUBLIC $<BUILD_INTERFACE:${OpenCV_LIBS}>)
```

Comment:
What benefits it can bring in?

Reply:
Remove opencv deps

<img width="776" alt="image" src="https://github.com/open-mmlab/mmdeploy/assets/16019484/de7ba252-c1ca-4bce-946e-7eabf381a87b">